_positions/epa-fellow-cloud-architect.md

```diff
@@ -0,0 +1,53 @@
+---
+title: EPA Innovation Fellow - Cloud Architect/Infrastructure Engineer
+permalink: raas/epa/cloud-architect/
+team: epa
+active: true
+description: Job posting for the EPA fellow role of Cloud Architect.
+---
+
+*Please Note: This role is for the Environmental Protection Agency's innovation fellowship program, not 18F.*
+
+#Cloud Architect/Infrastructure Engineer
+
+##Basic Information:
+
+**Agency:** [Environmental Protection Agency (EPA)](https://www3.epa.gov/)
+
+**Position Type:** Excepted Service - Time-Limited Appointment - Not to exceed 24 months; may be extended for a maximum of 4 years total
+
+**Compensation/Grade Level:** GS-14.GS-15
+- [Information regarding the General Schedule Grade Levels](https://www.opm.gov/policy-data-oversight/pay-leave/pay-systems/general-schedule/0)
+- [General Schedule Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/)
+
+**Key Requirements:** U.S. Citizenship is required.
+
+**Security Clearance:** Public Trust - Background Investigation Required
```

Comment:
Do we have any further information to which we can link people? Background investigations are unfamiliar to many people. I believe join.18f.gov includes some info; would it be the same?